qcodes/utils/installation_info.py

```diff
@@ -47,7 +53,18 @@ def get_qcodes_version() -> str:
     """
     Get the version of the currently installed QCoDeS
     """
-    return qcodes.__version__
+    package_name = "qcodes"
+
+    qcodes_path = files(package_name)
```

Comment:
This breaks since this does not give the path to the qcodes install folder but to the source if you run this from the checkout dir.